sdk/README.md

```diff
@@ -0,0 +1,40 @@
+# Compiler for Tekton
+
+As we know, there is a SDK of `kubeflow pipeline` for end user to define pipelines for machine learning(ML). Actually the output stuff fo the SDK is yaml for [Argo](https://github.com/argoproj/argo).
+
+In this place, we try to update the `Comiler` of SDK to generate `Tekton` stuff.
+
+## Development Prerequisites
+1. [`Python`](https://www.python.org/downloads/): Python 3.5 or later  
+2. [`Conda`](https://docs.conda.io/en/latest/): Package, dependency and environment management for Python
+
+
+## Take a try  
+
+1. Setup Python env with Conda.
+
+2. Clone the [Kubeflow Pipelines](https://github.com/kubeflow/pipelines)  
```

Comment:
KubeFlow Pipeline repository will change over time. Do you have a commit ID where this SDK is based on?

Reply:
Thanks for comments, I test code with latest master, could work, so I use the latest commit in master branch.